clang/unittests/Analysis/FlowSensitive/TransferTest.cpp

```diff
@@ -2313,6 +2313,42 @@ TEST(TransferTest, AssignmentOperatorWithInitAndInheritance) {
          ASTContext &ASTCtx) {});
 }
 
+TEST(TransferTest, CXXOperatorCallExprEqualReturnsVoid) {
+  // This is a crash repro.
+  std::string Code = R"(
+    struct B {
```

Comment:
Can you rename this to `struct S`?

(The reason we have `B` in the test above is because it is the "base". But otherwise we typically call structs `S`.)